src/sonic-yang-models/yang-models/sonic-static-route.yang

```diff
@@ -0,0 +1,100 @@
+module sonic-static-route {
+  yang-version 1.1;
+  namespace "http://github.com/Azure/sonic-static-route";
+  prefix sroute;
+
+  import sonic-vrf {
+    prefix vrf;
+  }
+  import ietf-inet-types {
+    prefix inet;
+  }
+  import sonic-types {
+    prefix stypes;
+  }
+
+  organization
+    "SONiC";
+  contact
+    "SONiC";
+  description
+    "STATIC ROUTE yang Module for SONiC OS";
+
+  revision 2022-03-17 {
+    description
+      "First Revision";
+  }
+
+  container sonic-static-route {
+    container STATIC_ROUTE {
+      description
+        "STATIC_ROUTE part of config_db.json";
+      list STATIC_ROUTE_LIST {
+        key "vrf_name prefix";
+        leaf prefix {
+          type stypes:sonic-ip4-prefix;

```

Comment:
We should support ipv6 prefix as well.

Reply:
Can add it here in yang, however would not be able to test it as a solution.